### PR TITLE
Bruk av db-scheduler

### PR DESCRIPTION
**Pull request by stianStensli**
Dette burde prioriteres opp da hovedindekeringen ikke kjører som forventet: https://logs.adeo.no/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15d,to:now))&_a=(columns:!(message,envclass,level,application,host),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:envclass,negate:!t,params:(query:q),type:phrase),query:(match_phrase:(envclass:q)))),index:'96e648c0-980a-11e9-830a-e17bbd64b4db',interval:auto,query:(language:lucene,query:hovedindeksering),sort:!())

### CODE REVIEW

**Comment by matiasvinjevoll** [comment]
Verdt å sjekke at central repo prioriteres over jitpack, ref. https://github.com/navikt/veilarbvedtaksstotte/pull/237/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R34

**Comment by matiasvinjevoll** [comment]
Kan leaderElection i `nais-[dev/prod].yaml` tas bort?

**Comment by stianStensli** [comment]
`Kan leaderElection i nais-[dev/prod].yaml tas bort?` Usikker, bruker ikke kafka common leaderElection?

File: pom.xml [hunk 1]
```diff
@@ -352,6 +357,11 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <repositories>
+                <!-- Legger til central eksplisitt for prioritet over jitpack -->
+                <repository>
+                    <id>central</id>
+                    <url>https://repo.maven.apache.org/maven2</url>
+                </repository>
```
Comment (stianStensli) [inline pom.xml:364]: Det er allerede en central repository, betyr det da egentlig at det er ok?: 
              <id>central</id>
              <url>https://repo1.maven.org/maven2</url>

**Pull request merged**